### PR TITLE
feat: add dark mode

### PR DESCRIPTION
**Pull request by sametpalitci**
Dark Mode:
dark

Light Mode:
light

### CODE REVIEW

File: src/generator/apitypes.ts [hunk 1]
```diff
@@ -80,10 +80,10 @@ class TypesGenerator implements Generatable<TypesGeneratorStructure> {
 
   toHTML() {
     return `<div>
-    <h1 class="text-3xl" id="types">Types</h1>
+    <h1 class="text-3xl dark:text-white" id="types">Types</h1>
         <div>
           <div class="ml-4">
-            <h3 class="mb-2 text-2xl font-normal" id="input-types">Input Types</h3>
+            <h3 class="mb-2 text-2xl font-normal" id="input-types dark:text-white">Input Types</h3>
```
Comment (pantharshit00) [inline src/generator/apitypes.ts:86]: Please do the same for output types on line 94. Right now it is of wrong color in dark mode

File: src/printer/index.ts [hunk 2]
```diff
@@ -59,17 +89,18 @@ export default class HTMLPrinter implements Generatable<DMMFDocument> {
   <head>
     ${this.getHead('styles/main.css')}
   </head>
-  <body class="bg-gray-200">
+  <body class="bg-gray-200 dark:bg-black">
     <div class="flex min-h-screen">
       <div
-        class="sticky top-0 w-1/5 flex-shrink-0 h-screen p-4 px-6 mr-4 overflow-auto bg-white mac-h-screen"
+        class="sticky top-0 w-1/5 flex-shrink-0 h-screen p-4 px-6 overflow-auto bg-white dark:bg-black mac-h-screen"
```
Comment (pantharshit00) [inline src/printer/index.ts:95]: I would like to have same bg color for sidebar and content so match them to gray-800

**Review by pantharshit00 — CHANGES_REQUESTED** [review]
Thanks for the work! I have requested a few small changes then it should be good.

Also, please update the snapshots once after you have made the changes using `npm run test -- -u`. Right now CI is failing because of that. 

Finally you can also maybe look into the code block sections and update those with dark theme using something like https://github.com/antfu/prism-theme-vars. That would be the final touch. You can also do a separate PR for that if you don't have enough time.

**Comment by sametpalitci** [comment]
I've made the requested changes and updated the snapshot. I'm considering maybe making the Prism theme in another pull request at a later time. 

Thank you.

**Review by pantharshit00 — APPROVED** [review]

**Review by pantharshit00 — APPROVED** [review]

**Pull request merged**